Cargo.toml

```diff
@@ -27,7 +27,7 @@ semver = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 snafu = "0.8"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.1" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", branch = "main" }
```

Comment:
This needs to point to a released version before this PR can be merged.

Reply:
Also, it is recommended to use the patch mechanism instead of adjusting the dependency directly.